README.md

```diff
@@ -1,3 +1,3 @@
 This plugin allows to define build environment as a Dockerfile, stored in project SCM.
-see also http://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Docker+Custom+Build+Environment+Plugin
+see also http://wiki.jenkins-ci.org/display/JENKINS/Docker+Custom+Build+Environment+Plugin
```

Comment:
Should not be merged unless and until the new page is created and old deprecated with a redirect.